lib/mappings/walletInfo.js

```diff
@@ -0,0 +1,84 @@
+import {
+  selectWallet,
+  selectAccount,
+  createAccount,
+  getAccountHistory,
+} from '../actions';
+
+import {
+  selectWalletInfo,
+  selectMultisigWalletInfo,
+  selectSelectedWallet,
+  selectSelectedAccount,
+  selectWalletAccounts,
+  selectWalletAccountInfo,
+  selectHistory,
+} from './selectors';
+
+// mapped to constants/WALLET_INFO_ACCOUNTS_LIST_HEADERS
```

Comment:
`bwallet/constants/...`?

Reply:
what is mapped to the constants here? Not following the comment.

Reply:
Let me know if the new comment doesn't clear this up